app/templates/network/show/details.hbs

```diff
@@ -4,19 +4,28 @@
   <div id="details-sidebar" class="{{if networkInput "network-settings-open"}}">
     <div class="sidebar-header">
       <h4 class="network-title">
-        {{!-- TODO: this is a button for showing the side nav once ready
-        <a class="sidebar-nav-toggle" href="#"><i class="fa fa-bars"></i></a>  --}}
-          {{#if model.name}}
+        <a href="#" class="navbar-toggle"><i class="material-icons">menu</i></a>
+        <script type="application/javascript">
```

Comment:
I will move this to the controller, in an afterRender hook. We should avoid putting scripts in templates.